src/Control/Walker/CmdLine/CmdLine.h

```diff
@@ -83,6 +83,7 @@ class CmdLine : public tk::Control<
     //!   contains a copy of the command-line stack, the command-line stack must
     //!   be possible to be instantiated without passing the ctrinfo map,
     //!   otherwise it would be a mutual dependency.
+    // cppcheck-suppress noExplicitConstructor
```

Comment:
I wonder if we can have some standard comment that the parsers still recognize that quickly tells human it's an annotation? 

`//@@ cppcheck-suppress noExplicitConstructor`

After thinking a bit more, I guess given a lot of the compiler ones are pragmas it's harder